website/templates/konfig.php

```diff
@@ -1,63 +1,69 @@
 <?php
 
 // config file for GeoKrety
-$config['prod_server_name'] = 'geokrety.org';
-$config['adres'] = 'https://geokrety.org/';
+$config['prod_server_name'] = isset($_ENV['PROD_SERVER_NAME']) ? $_ENV['PROD_SERVER_NAME'] : 'geokrety.org';
+$config['adres'] = isset($_ENV['PROD_SERVER_URL']) ? $_ENV['PROD_SERVER_URL'] : $config['prod_server_name'];
```

Comment:
(minor) we could add a method to factorize the way we are defining a given config:
````
fromEnvOrDefault('PROD_SERVER_URL',$config['prod_server_name']);
````

Reply:
I would have loved to use the new "[Null coalescing operator](https://www.php.net/manual/en/migration70.new-features.php)", but we're still using `php 5` which is BTW [not supported anymore](https://www.php.net/supported-versions.php) :(




> The null coalescing operator (??) has been added as syntactic sugar for the common case of needing to use a ternary in conjunction with isset(). It returns its first operand if it exists and is not NULL; otherwise it returns its second operand."

```
// Fetches the value of $_GET['user'] and returns 'nobody'
// if it does not exist.
$username = $_GET['user'] ?? 'nobody';
// This is equivalent to:
$username = isset($_GET['user']) ? $_GET['user'] : 'nobody';
```

Reply:
so you mean that we need a new task to migrate to php7 :)

Reply:
In an ideal world it would be nice.

Reply:
> we could add a method to factorize the way we are defining a given config

Let's postpone this until we migrate to php 7